libbeat/idxmgmt/std.go

```diff
@@ -360,6 +364,16 @@ func getEventCustomIndex(evt *beat.Event) string {
 		}
 	}
 
+	// This is functionally identical to Meta["alias"], returning the overriding
+	// metadata as the index name if present. It is currently used by Filebeat
+	// to send the index for particular inputs to formatted string templates,
+	// which are then expanded by a processor to the "raw-index" field.
+	if tmp := evt.Meta["raw-index"]; tmp != nil {
+		if idx, ok := tmp.(string); ok {
+			return idx
+		}
+	}
```

Comment:
hm... where is alias used? The name looks very specific to some use-case. I guess it was added to work around the timestamp addition when `index` is used. Should we go with 'alias' or 'raw-index' in general?

Reply:
I don't know who might be using `alias` or how, which is why I was reluctant to overload it (that, and because of the potentially confusing name)... my preference is for `raw-index` or something similar (`raw-index-name`?) since the functional behavior is independent of whether aliases are being used, though I could be convinced to go with `alias` instead to avoid proliferating magic Metadata fields...

Reply:
Looks like we need to grep the source. I guess this comes from Metricbeat or Heartbeat.

Reply:
this is odd I don't remember that?

Reply:
I think I copied it from the original ILM PR. It was introduced in 7.0. Not sure if anyone still uses it. Removing could break things. We should at least create a follow up issue (technical debt) to investigate and remove it in the future.